### PR TITLE
Why are you always resolving

### PR DESCRIPTION
**Pull request by permcody**
refs #11407

### CODE REVIEW

File: framework/src/base/MooseApp.C [hunk 1]
```diff
@@ -1320,48 +1320,51 @@ MooseApp::getMeshModifierNames() const
 void
 MooseApp::executeMeshModifiers()
 {
-  DependencyResolver<std::shared_ptr<MeshModifier>> resolver;
-
-  // Add all of the dependencies into the resolver and sort them
-  for (const auto & it : _mesh_modifiers)
+  if (!_mesh_modifiers.empty())
```
Comment (permcody) [inline framework/src/base/MooseApp.C]: This line is a change

File: framework/src/problems/FEProblemBase.C [hunk 2]
```diff
@@ -2977,44 +2977,47 @@ FEProblemBase::computeUserObjects(const ExecFlagType & type, const Moose::AuxGro
 void
 FEProblemBase::executeControls(const ExecFlagType & exec_type)
 {
-  DependencyResolver<std::shared_ptr<Control>> resolver;
-
-  auto controls_wh = _control_warehouse[exec_type];
-  // Add all of the dependencies into the resolver and sort them
-  for (const auto & it : controls_wh.getActiveObjects())
+  if (_control_warehouse[exec_type].hasActiveObjects())
```
Comment (permcody) [inline framework/src/problems/FEProblemBase.C]: This line is a change

**Comment by permcody** [comment]
Note to reviewer: This is actually a 2 line change - use the filter whitespace viewer.

**Review by friedmud — APPROVED** [review]
👍

**Pull request merged**